### PR TITLE
Re-enable survey prompt

### PR DESCRIPTION
**Pull request by nkubala**
fixes https://github.com/GoogleContainerTools/skaffold/issues/4630

also, slightly change wording and add a bit of color.

### CODE REVIEW

**Review by tstromberg — DISMISSED** [review]
Which commands will this be enabled for? I want to make sure that it's only for a specific set of white-listed commands where it makes the most sense to show, rather than "help" and "version".

**Comment by tejal29** [comment]
> Which commands will this be enabled for? I want to make sure that it's only for a specific set of white-listed commands where it makes the most sense to show, rather than "help" and "version".

Currently, we are enabling this for all commands.

**Comment by nkubala** [comment]
> Which commands will this be enabled for? I want to make sure that it's only for a specific set of white-listed commands where it makes the most sense to show, rather than "help" and "version".

this is a good point, showing the survey text for `help` really makes no sense. I think we probably only want to show it for "interactive" subcommands - `dev`, `debug`, `run`, `build`, `deploy`, maybe `render`. whereas it might not make sense on `init`, and definitely doesn't make sense on `credits` or `version`.

i'll update this PR and list out the commands it's enabled for once i do.

**Comment by nkubala** [comment]
whoops, totally forgot we added this already: https://github.com/GoogleContainerTools/skaffold/pull/4622

fixing the tests then this should be ready

File: pkg/skaffold/survey/survey.go [hunk 1]
```diff
@@ -22,17 +22,17 @@ import (
...
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/config"
-	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/version"
 )
 
 const (
-	Prompt = `Help improve Skaffold! Take a 10-second anonymous survey by running
-   skaffold survey`
+	Prompt = `Help us make Skaffold better! Take our quick and anonymous survey by running
+   'skaffold survey'
+`
```
Comment (briandealwis) [inline pkg/skaffold/survey/survey.go:35]: I got this to 79 characters with:
```
Help improve Skaffold with our 2-minute anonymous survey: run `skaffold survey`
```

**Review by briandealwis — APPROVED** [review]

File: pkg/skaffold/survey/survey.go (hunk 1 above)
Comment (nkubala) [reply pkg/skaffold/survey/survey.go]: updated, thanks

**Pull request merged**